packages/sources/alongside/src/endpoint/collateral/index.ts

```diff
@@ -59,27 +60,32 @@ const sign = (str: string, secret: string) => {
   return hash.toString(CryptoJS.enc.Base64)
 }
 
-export class AlongsideCollateralTransport implements Transport<EndpointTypes> {
-  // Global variable to keep the token. Token is provisioned when the accounts endpoint is hit.
-  // Each instance of the EA will have their own token by design
-  token!: string
+export class AlongsideCollateralTransport extends SubscriptionTransport<EndpointTypes> {
   cache!: Cache<AdapterResponse<EndpointTypes['Response']>>
-  responseCache!: ResponseCache<any>
+  responseCache!: ResponseCache<{
+    Request: EndpointTypes['Request']
+    Response: EndpointTypes['Response']
+  }>
   requester!: Requester
   name!: string
 
   async initialize(
     dependencies: TransportDependencies<EndpointTypes>,
-    _: typeof config.settings,
-    __: string,
+    adapterSettings: typeof config.settings,
+    endpointName: string,
     name: string,
   ): Promise<void> {
+    super.initialize(dependencies, adapterSettings, endpointName, name)
     this.cache = dependencies.cache as Cache<AdapterResponse<EndpointTypes['Response']>>
```

Comment:
This should be removed, the cache is not used now that the responseCache is preferred